mappings/net/minecraft/recipe/CookingCategory.mapping

```
@@ -1,5 +0,0 @@
-CLASS net/minecraft/unmapped/C_gtxamkec net/minecraft/recipe/CookingCategory
```

why?

---

The class was declared in two places, so I deleted the one that wasn't being added to the main mappings file

---

That should probably throw an error, or at least a warning